controller/array_action/array_mediator_ds8k.py

```diff
@@ -384,8 +384,14 @@ def get_volume_name(self, volume_id):
         logger.debug("Searching for volume with id: {0}".format(volume_id))
         try:
             api_volume = self.client.get_volume(volume_id)
```

Comment:
can you unify the `self.client.get_volume(volume_id)` calls?

Reply:
what do you mean?
should calling `get_volume_name` invoke `get_flashcopies_by_volume`?

Reply:
anyway, I'm not sure if it's relevant to this PR, which is already large

Reply:
`get_flashcopies_by_volume ` is not an actual operation, it's just a workaround to a  bug with pyds8k so It should be called only where necessary and not in the common method. I can see that it will cause another `try` block and eventually won't save lines.. forget about it for now..